operating/manage-security-policies.adoc

```diff
@@ -22,8 +23,8 @@ include::modules/modify-existing-security-policies.adoc[leveloffset=+1]
 
 You can create and manage policy categories by using the following methods:
 
-* Create a policy category in the *Policy details* section when you create a policy. The category names are stored as strings that are attached to the policy and cannot be copied or deleted.
-* Create, copy, or delete a policy category by navigating to *Platform Configuration* -> *Policy Management* and clicking the *Policy categories* tab. The policy categories are stored in the database and can be managed. This option is available only in {product-title-managed} or in {product-title-short} if you have the PostgreSQL database (Technology Preview) enabled.
+* Create a policy category in the *Policy details* section when you create a policy. The category names are stored as strings that are attached to the policy and cannot be copied or deleted. This option is only available in {product-title-short} release 3.74 and earlier.
```

Comment:
categories can be deleted even today

Reply:
So this seems to be only available in pre-3.74 (ability to add categories when creating a policy from the Policy details screen). I need to leave this in for older versions and yet direct people post 4.0 to use the new tab.

Reply:
Yes this is no longer applicable in 4.0+